example/cumulus-tf/variables.tf

```diff
@@ -483,9 +483,11 @@ variable "workflow_configurations" {
 
     - `sf_event_sqs_to_db_records_types`: An optional nested map that controls which record types
       ("execution", "granule", "pdr") should be written to the database for each workflow and
-      workflow status("running", "completed", "failed").
+      workflow status ("running", "completed", "failed").
       This configuration is used by the `@cumulus/api/sfEventSqsToDbRecords` Lambda.
 
+      Currently, both "execution" and "pdr" must be written to the database, so the record type list must include both.
+
```

Comment:
Below, on line 514 we're saying the default is `running = ["execution", "pdr"]` but shouldn't that be an empty object? Or `running = ["execution", "pdr", "granule"]`?

Reply:
Don't we want the example deployment to be `sf_event_sqs_to_db_records_types = {}`? Sorry, I may just be misunderstanding.